LibVLCSharp.Forms/Shared/PlaybackControls.xaml.cs

```diff
@@ -0,0 +1,1369 @@
+                    }
+
+                    var var = (double)videoWidth / videoHeight;
+                    var scalingFactor = Device.Info.ScalingFactor;
+                    var screenWidth = videoView.Width * scalingFactor;
+                    var screenHeight = videoView.Height * scalingFactor;
+                    var screenar = screenWidth / screenHeight;
+                    mediaPlayer.Scale = (float)(screenar >= var ? screenWidth / videoWidth : screenHeight / videoHeight);
+                }
+                else
+                {
+                    mediaPlayer.Scale = 0;
+                }
+            }
+            catch (Exception ex)
+            {
+                ShowErrorMessageBox(ex);
+            }
+        }
+
+        private void SeekBar_ValueChanged(object sender, ValueChangedEventArgs e)
+        {
+            _ = FadeInAsync();
+
+            UpdateTime();
+            SeekBarTimerEnabled = true;
+            SeekBarTimer.Change(TimeSpan.FromSeconds(1), TimeSpan.FromMilliseconds(-1));
+        }
+
+        private void UpdateMediaPlayerPosition()
+        {
+            try
+            {
+                var mediaPlayer = MediaPlayer;
+                if (mediaPlayer != null)
+                {
+                    mediaPlayer.Position = (float)(SeekBar.Value / SeekBar.Maximum);
+                }
+            }
+            catch (Exception ex)
+            {
+                ShowErrorMessageBox(ex);
+            }
+            finally
+            {
+                SeekBarTimerEnabled = false;
+            }
+        }
+
+        private void UpdatePosition(float? position = null)
+        {
+            try
+            {
+                var mediaPlayer = MediaPlayer;
+                var pos = position ?? mediaPlayer?.Position ?? 0;
+                var timeSpanPosition = TimeSpan.FromMilliseconds((mediaPlayer?.Length ?? 0) * pos);
+                var elapsedTime = (timeSpanPosition - Position).TotalMilliseconds;
+                if (position != null && elapsedTime > 0 && elapsedTime < 750)
+                {
+                    return;
+                }
+
+                UpdateSeekBar(timeSpanPosition, pos);
+            }
+            catch (Exception ex)
+            {
+                ShowErrorMessageBox(ex);
+            }
+        }
+
+        private void UpdateTime(double? position = null)
+        {
+            if (RemainingTimeLabel == null)
+            {
+                return;
+            }
+
+            try
+            {
+                var mediaPlayer = MediaPlayer;
+                var state = mediaPlayer?.State;
+                var length = mediaPlayer == null || state == VLCState.Ended || state == VLCState.Error || state == VLCState.NothingSpecial ||
+                    state == VLCState.Stopped ? 0 : mediaPlayer.Length;
+                var time = position == null ? (SeekBar.Value * length / SeekBar.Maximum) : (double)position;
```

Comment:
This is equivalent to:

```suggestion
                var time = position ?? (SeekBar.Value * length / SeekBar.Maximum);
```